### ISSUE TITLE
Storybook doesn't import theme properly

### ISSUE THREAD

**Issue by stefanalund** [spotify/backstage#530]
The Backstage theme is not reflected in the Storybook, making the components look off. Components that try to import the theme even fails:

<img width="1108" alt="Screenshot 2020-04-09 at 15 02 18" src="https://user-images.githubusercontent.com/190856/78898210-b97c9a00-7a73-11ea-92bc-272ff2f93d21.png">

## Expected Behavior

Components should render identically in Storybook as in normal Backstage app. 
## Steps to Reproduce (for bugs)

Run Storybook locally according to docs here https://github.com/spotify/backstage/blob/master/docs/getting-started/contributing-to-storybook.md

## Context

I don't think has ever worked.

## Your Environment

* NodeJS Version (v12):
* Operating System and Version (e.g. Ubuntu 14.04):
* Browser Information:

**Comment by stefanalund** [comment]
Fixed in #539

**Comment by stefanalund** [comment]
Another indicator that this is still a problem: The typeface is still not correct in http://storybook.backstage.io/?path=/story/progress-card--default

**Comment by stefanalund** [comment]
cc @mikqi @OrkoHunter

**Comment by mikqi** [comment]
I try to add decorator globally to fix this issue, but the custom themes is not exist only default value. And then i try to add decorator in `stories`. The custom theme exist.
![image](https://user-images.githubusercontent.com/4416419/79227827-b4ac5180-7e8a-11ea-9b41-3e107ec727ab.png)

so the code will look like this, or we can make decorator as a utility. any better idea?
```js
export default {
  title: 'Status',
  component: StatusOK,
  decorators: [
    storyFn => (
      <ThemeProvider theme={BackstageTheme}>
        <CssBaseline>{storyFn()}</CssBaseline>
      </ThemeProvider>
    ),
  ],
};
```

**Comment by stefanalund** [comment]
NICE! The optimal setup would be if you wouldn't have to add this to every single story. But if that's not viable this looks like a great (stop-gap) solution.

**Comment by hooloovooo** [comment]
@mikqi Is this something like what you were suggesting?
https://github.com/spotify/backstage/pull/556
This should solve it globally and remove the need to wrap the stories individually, but it might also cause issues(?).

**Comment by mikqi** [comment]
> @mikqi Is this something like what you were suggesting?
> #556
> This should solve it globally and remove the need to wrap the stories individually, but it might also cause issues(?).

@hooloovooo have you try to add this [story](https://github.com/spotify/backstage/pull/540/files#diff-da269469440e098ca43f01168859f968R33)? i have try to add it globaly, but the custom theme also not exists. after that i try to add decorator in stories file and the custom theme variable exists.

**Comment by mikqi** [comment]
> @mikqi Not exactly sure what you mean and what to expect it to look like, but it's there and works when i add it.
> 
> ![2020-04-15-163149_981x821_scrot](https://user-images.githubusercontent.com/701797/79349550-a0259300-7f36-11ea-8f67-64bd92c0537d.png)

Weird. I did the same thing in my laptop. but it didn't work. 😮

**Comment by hooloovooo** [comment]
> 
> 
> > @mikqi Not exactly sure what you mean and what to expect it to look like, but it's there and works when i add it.
> > ![2020-04-15-163149_981x821_scrot](https://user-images.githubusercontent.com/701797/79349550-a0259300-7f36-11ea-8f67-64bd92c0537d.png)
> 
> Weird. I did the same thing in my laptop. but it didn't work. 😮

That's weird. Maybe it was just the build that didn't pick up the new config file or something.

**Comment by stefanalund** [comment]
This is fixed in #556 - thanks everyone for helping out.
